src/utils/TedexSetup.ts

```diff
@@ -0,0 +1,59 @@
+import { TraderClient, TradeType } from 'tdex-sdk';
+import { toSatoshi } from './format';
+import BigNumber from 'bignumber.js';
+import { Market } from './constants';
+
+const client = new TraderClient('http://localhost:9945');
+
+export const handleMarkets = async () => {
+  try {
+    const markets = await client.markets();
+    console.log(markets);
+
+    return markets;
+  } catch (error) {
+    throw new Error('TDEX provider and markets is not reachable');
+  }
+};
+
+export const onSendAmountChange = async (
+  amount: number,
+  trade: TradeType,
+  asset: string,
+) => {
+  try {
+    const pricesPreview = await client.marketPrice(
+      {
+        baseAsset: Market.L_BTC,
```

Comment:
here you are fixing always L_BTC and L_USDT. This method should pass as parameter the trading pair selected, which we know from user selection.

Reply:
fixed this

Reply:
fixed this